docs/user_workflow.adoc

```diff
@@ -0,0 +1,90 @@
++--------------------------------------+--------+--------+------------+-------------+-------------------------------------------------------+------------+--------------------------------------+----------+-------------------+-----------------+------------+-------------+
+| 01892b9b-c7bc-4e02-8132-9ca085f875d3 | test_7 | ACTIVE | None       | Running     | private_demo=${NETWORK_PREFIX}.122.205, 192.168.0.131 | cirros     | 2947bd5b-2cc8-4d6f-8347-8d571665aab7 | m1.small | nova              | mytest-compute1 |            | UP          |
+| 42fbbdd6-c605-4c4f-8918-7e629003b702 | test_6 | ACTIVE | None       | Running     | private_demo=${NETWORK_PREFIX}.122.239, 192.168.0.130 | cirros     | 2947bd5b-2cc8-4d6f-8347-8d571665aab7 | m1.small | nova              | mytest-compute2 |            | UP          |
+| a441b8d2-f501-44b3-858f-187bca02998b | test_5 | ACTIVE | None       | Running     | private_demo=${NETWORK_PREFIX}.122.218, 192.168.0.125 | cirros     | 2947bd5b-2cc8-4d6f-8347-8d571665aab7 | m1.small | nova              | mytest-compute1 |            | UP          |
+| 9b69cf58-d8ce-4642-9ea3-37216983e3a5 | test_4 | ACTIVE | None       | Running     | private_demo=${NETWORK_PREFIX}.122.201, 192.168.0.68  | cirros     | 2947bd5b-2cc8-4d6f-8347-8d571665aab7 | m1.small | nova              | mytest-compute2 |            | UP          |
+| f3c557fe-676c-4528-b972-b70385e55407 | test_3 | ACTIVE | None       | Running     | private_demo=${NETWORK_PREFIX}.122.179, 192.168.0.41  | cirros     | 2947bd5b-2cc8-4d6f-8347-8d571665aab7 | m1.small | nova              | mytest-compute1 |            | UP          |
+| 2f30da53-cd8f-441c-b1e8-e5e074fffa14 | test_2 | ACTIVE | None       | Running     | private_demo=${NETWORK_PREFIX}.122.188, 192.168.0.95  | cirros     | 2947bd5b-2cc8-4d6f-8347-8d571665aab7 | m1.small | nova              | mytest-compute2 |            | UP          |
+| b6a17b99-2d8c-4551-a91c-9dfe0282e886 | test_1 | ACTIVE | None       | Running     | private_demo=${NETWORK_PREFIX}.122.204, 192.168.0.127 | cirros     | 2947bd5b-2cc8-4d6f-8347-8d571665aab7 | m1.small | nova              | mytest-compute1 |            | UP          |
+| 25a7667e-ab3e-43a8-b6d0-de9e02777530 | test_0 | ACTIVE | None       | Running     | private_demo=${NETWORK_PREFIX}.122.202, 192.168.0.155 | cirros     | 2947bd5b-2cc8-4d6f-8347-8d571665aab7 | m1.small | nova              | mytest-compute1 |            | UP          |
++--------------------------------------+--------+--------+------------+-------------+-------------------------------------------------------+------------+--------------------------------------+----------+-------------------+-----------------+------------+-------------+
+----
+
+== Watcher Workflow (CLI)
+=== Optimization Scenario I: Instances Consolidation in Minimal Compute Nodes ===
+
```

Comment:
may be nice to link to the strategy doc for more info on parameters https://docs.openstack.org/watcher/2024.2/strategies/node_resource_consolidation.html#configuration